plugins/modules/route53_health_check.py

```diff
@@ -594,13 +594,6 @@ def main():
     action = None
     check_id = None
 
-    if module.params.get('use_unique_names') or module.params.get('health_check_id'):
-        module.deprecate(
-            'The health_check_name is currently non required parameter.'
-            ' This behavior will change and health_check_name '
-            ' will change to required=True and use_unique_names will change to default=True in release 6.0.0.',
```

Comment:
>             ' will change to required=True and use_unique_names will change to default=True in release 6.0.0.',

L531 and L532 are not changed to [`required/default=True`](https://github.com/ansible-collections/amazon.aws/blob/main/plugins/modules/route53_health_check.py#L531-L532) yet.

Reply:
@mandar242 ^

Reply:
Ah sorry. I just looked at the code change, whitch itself without context is not sufficient.  
lgtm